app/views/layouts/application.html.erb

```diff
@@ -11,6 +11,8 @@
     <%= javascript_pack_tag 'application' %>
     <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Poppins:300,400,500,700&display=swap" />
     <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons" />
+    <link href="/assets/favicon.ico" rel="shortcut icon" type="image/x-icon" />
```

Comment:
Copied from the rails docs. Is there a better location to keep this than `/assets`?

Reply:
It looks like maybe you can use `favicon_link_tag `? https://medium.com/@josephcardillo/how-to-add-a-favicon-to-your-rails-app-9676336f7006

Reply:
Done 👍